Desktop/utilities/qmlutils.cpp

```diff
@@ -31,3 +31,51 @@ QJSValue	QmlUtils::decodeJson(const QJSValue	& val, QQuickItem * caller)
 	ColumnEncoder::decodeJson(v);
 	return tqj(v, caller);
 }
+
+
+#ifdef linux
+
+#include <QtGlobal>
```

Comment:
I would prefer it if the includes were at the top to be honest.
That way it is a bit simpler to figure out what is being included when you open a file.